### PR TITLE
Adding settings to GitHub Api

### PR DESCRIPTION
**Pull request by carterm**
Moving github settings away from module so we can target different repos

### CODE REVIEW

**Comment by carterm** [comment]
Draft until I have more time to test it.

File: CovidWeeklyTierUpdate/doUpdate.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ const prepData = async () => {
 
 //Check to see if we need stats update PRs, make them if we do.
 const doWeeklyUpdatePrs = async mergetargets => {
+    gitHubSetConfig('cagov','covid',process.env["GITHUB_TOKEN"],process.env["GITHUB_NAME"],process.env["GITHUB_EMAIL"]);
```
Comment (aaronhans) [inline CovidWeeklyTierUpdate/doUpdate.js:34]: should the repo be covid19 here?

Comment (carterm) [reply CovidWeeklyTierUpdate/doUpdate.js:34]: yes...checking

Comment (carterm) [reply CovidWeeklyTierUpdate/doUpdate.js]: It was a long week.

**Review by aaronhans — APPROVED** [review]

**Pull request merged**